cool_seq_tool/sources/uta_database.py

```diff
@@ -865,8 +865,8 @@ async def get_gene_from_ac(
 
     async def get_transcripts_from_gene(
         self,
-        start_pos: int,
-        end_pos: int,
+        start_pos: int = None,
+        end_pos: int = None,
         gene: Optional[str] = None,
         use_tx_pos: bool = True,
         alt_ac: Optional[str] = None,
```

Comment:
Make sure to update the docstring to describe intended function -- ie that it'll fetch all associated transcripts if start/end aren't specified